src/Core/Checkout/Order/OrderDefinition.php

```diff
@@ -106,6 +106,7 @@ protected function defineFields(): FieldCollection
             (new StringField('affiliate_code', 'affiliateCode'))->addFlags(new ApiAware()),
             (new StringField('campaign_code', 'campaignCode'))->addFlags(new ApiAware()),
             (new LongTextField('customer_comment', 'customerComment'))->addFlags(new ApiAware(), new AllowEmptyString()),
+            (new LongTextField('internal_comment', 'internalComment'))->addFlags(new ApiAware(), new AllowEmptyString()),
```

Comment:
if we want to stand by the "privacy" claim in the snippet, should we maybe take `ApiAware` away to have it only available in the Admin API?